extensions/devservices/common/src/main/java/io/quarkus/devservices/common/ConfigureUtil.java

```diff
@@ -38,4 +42,27 @@ public static String configureSharedNetwork(GenericContainer<?> container, Strin
 
         return hostName;
     }
+
+    public static String getDefaultImageNameFor(String devserviceName) {
+        var imageName = LazyProperties.INSTANCE.getProperty(devserviceName + ".image");
+        if (imageName == null) {
+            throw new IllegalArgumentException("No default image configured for " + devserviceName);
+        }
+        return imageName;
+    }
+
+    private static class LazyProperties {
+
+        private static final Properties INSTANCE;
+
+        static {
+            var tccl = Thread.currentThread().getContextClassLoader();
+            try (InputStream in = tccl.getResourceAsStream("devservices.properties")) {
```

Comment:
Not sure about "external" dev services. To cover those as well here we could load _all_ properties on the CP.
But then again this can be considered more of an "internal" feature for the beginning?